docs/src/dev-docs/tooling-containers.md

```diff
@@ -139,6 +139,20 @@ An image containing the CLP package.
   tools/docker-images/clp-package
   ```
 
+### Default Configurations
+
+While not intended for direct end-user deployment, the container includes these configurations for
+debugging purposes:
+
+| Setting         | Related Env Var   | Value                                                  | Purpose                                          |
```

Comment:
I think some of these deserve more explanation. I'll come back to this PR after the release.